sandbox/extract-imagery-metadata_stevendepaschalis.R

```diff
@@ -49,56 +28,73 @@ extract_metadata = function(exif) {
   # Remove NA and infinity values
   time_diff_seconds <- time_diff_seconds[is.finite(time_diff_seconds)]
 
-  # Calculate image frequency (images per second)
-  image_frequency <- 1 / time_diff_seconds
+  # Calculate median and median absolute deviation (MAD)
+  median_time_diff <- median(time_diff_seconds)
+  mad_time_diff <- mad(time_diff_seconds)
+
+  # Set threshold as median + 3 * MAD
+  threshold <- median_time_diff + 3 * mad_time_diff
```

Comment:
What's the logic for using 3*MAD? I'm nor familiar with `mad`, and I'd be more comfortable with using quantiles as in my previous example unless you want to lobby for `mad`.